frame/proxy/src/lib.rs

```diff
@@ -178,6 +178,28 @@ pub mod pallet {
 		type AnnouncementDepositFactor: Get<BalanceOf<Self>>;
 	}
 
+	/// A trait for removing all proxies.
+	pub trait RemoveAllProxies {
```

Comment:
I don't think the issue required you to create a new trait? Simply making the `remove_all_proxy_delegates` public in the pallet is enough.

Reply:
Okay I make the change, earlier I was advised that it might require to implement a new trait.

Reply:
The point was to enable other pallets to call the function. Making it `pub` is enough, no need for another trait.